src/client.js

```diff
@@ -49,7 +48,7 @@ export class Client {
    */
   send(req) {
     const promise = new Promise((resolve, reject) => {
-      const callback = () => this.dispatch(req).then(resolve).catch(reject).finally(() => {
```

Comment:
What is the reason for this change?

Reply:
Moved from ES5 to ES6. `.finally` is not available on promises:

```
error TS2550: Property 'finally' does not exist on type 'Promise<void>'
```